magento/magento2
# Update Ga.php to fix no revenue data in GA4

### Description (*)

Fix for No revenue data in GA4

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/magento2/issues/37605

### Manual testing scenarios (*)

1. Switch on Google Analytics 4 part of the Magento Google Gtag extension
2. Place an order
3. Wait for data to appear in Google Analytics 4
4. On GA4, goto Reports->Monetization-> Verify the Revenue in the Monetization overview.

Correct revenue data is reported

### Questions or comments


### Contribution checklist (*)
